### DISCUSSION TITLE
Broken navigation on Samsung TV browser (Photos view, Explorer view) - viewing photos or getting back do not work

### DISCUSSION THREAD

**Discussion by dgtal1** [immich-app/immich#8890] [Feature Request]
### The bug

Not sure why, but the navigation is broken on my own immich instance, when I browse on my Samsung TV web browser.
See this video:
https://youtu.be/0kUsWPL0VnY

I'm pretty sure it worked initially, but stopped. Restarting my TV, refreshing the page doesn't solve it. 

**I've just noticed the same behavious happens to the immich demo! Initially all is working fine, but when I click the (i) icon when having the photo on full screen - the navigation gets broken.**

Interesting is that when I reopen immich in incognito mode it works again! This must be then related to storage or cookies? Not an expert here.

I love immich. It's got huge potential and it looks like it will be my ultimate photo solution that will replace Google Photos and Moments on my Synology NAS, but it must work properly with my TV! Photoprism for example works fully correct on my TV, but its AI object and facial recognition are bad and inaccurate and this feature in turn works flawlessly in immich.

Can someone please make immich Samsung TV compatible?

### The OS that Immich Server is running on

Windows + Ubuntu WSL

### Version of Immich Server

v1.101.0

### Version of Immich Mobile App

not using

### Platform with the issue

- [ ] Server
- [X] Web
- [ ] Mobile

### Your docker-compose.yml content

```YAML
I guess it's not relevant - same issue with immich demo
```

### Your .env content

```Shell
I guess it's not relevant - same issue with immich demo
```

### Reproduction steps

```bash
1. opent he Explore view and choos some category to get the thumbs - https://demo.immich.app/explore
2. try to click a thumbnail - nothing happens
3. click the back arrow - nothing happens
4. reopen immich in incognito mode
5. navigation works again until I click the (i) icon - broken again
...
```

**Comment by mertalev** [comment]
Sorry, but we don't currently officially support TV platforms. Using immich on these devices can break at any time as they're not part of our automated and manual testing. This may change in the future, but until then we don't recommend using immich on TV.

**Comment by dgtal1** [comment]
Oh, that's a pity :( but I understand. This is a showstopper for me then and a no-go. 
For me browsing photos on my TV is a must and a reason why I'm seeking for a good photo indexing solution working on my TV, Synology Moments is compatible, but truly sluggish. Photoprism runs quite well, but its AI facial recognition is bad. 

I guess I'm gonna be stuck with Photoprism then. Hope this changes in the future
